istio/oidc-authservice/base_v3/kustomization.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../base/service.yaml
+- ../base/statefulset.yaml
+- ../base/envoy-filter.yaml
+- ../base/pvc.yaml
```

Comment:
probably overlay/application.yaml is needed as well. I think besides `param -> config`, there's no need to migrate to v3 for this one. Can user use `overlay/applications.yaml` directly?

Reply:
> param -> config

Yes this is the main reason I want to migrate to V3. 

> Can user use overlay/applications.yaml directly?

Yes they can.

I'm okay with not migrating these components to V3. I think it might cause some incompatibility issues in the future. Since vendors (AWS/GCP) have moved to V3, and use `config` as the routine. 

For example, https://github.com/kubeflow/manifests/blob/master/stacks/aws/application/istio/kustomization.yaml#L7-L10

if we don't migrate to V3, these configMap will be `XX-parameters`, what do you think?

```
- name: istio-parameters
  behavior: merge
  envs:
  - params.env
```